build/simp-adapter.spec

```diff
@@ -83,59 +102,18 @@ install -p -m 640 -D src/conf/adapter_config.yaml %{buildroot}%{prefix}/adapter_
 #  - https://docs.puppet.com/pe/2016.4/install_what_and_where.html#group-accounts-installed
 #
 %defattr(-,root,root,-)
-%config(noreplace) %{prefix}/adapter_config.yaml
+%config(noreplace) %{prefix}/adapter_conf.yaml
```

Comment:
I think that this may leave an old copy of the previous config file if the user has modified it which may cause confusion.

Can we change this to a `%pre` section that removes the file if present and keep the name the same?

Reply:
I'll test the behavior to see what happens.  I didn't think noreplace was equivalent to 'no remove'.  Also, WRT to removing in the %pre and keeping the name the same, I think that would screw of RPM's bookkeeping.

Reply:
upon upgrade of simp-adapter 0.1.1 to 1.0.0 when local adapter_config.yaml had local modifications:

> el6 10:03:09$ ls -al /etc/simp
>  total 16
>  drwxr-xr-x.  2 root root 4096 Apr  4 14:03 .
>  drwxr-xr-x. 84 root root 4096 Apr  4 14:01 ..
>  -rw-r-----.  1 root root   68 Apr  4 14:03 adapter_config.yaml.rpmsave
>  -rw-r-----.  1 root root 1057 Apr  2 19:31 adapter_conf.yaml

I should be able to remove adapter_config.yaml.rpmsave in the simp-adapter %posttrans without any ill effects.  I'll verify.

Reply:
@lnemsick-simp `%posttrans` makes sense. I took a quick look through the docs and we don't reference it so it shouldn't creep beyond this component.